src/main/java/io/antmedia/muxer/MuxAdaptor.java

```diff
@@ -324,6 +331,9 @@ protected void enableSettings() {
 		previewHeight = appSettingsLocal.getPreviewHeight();
 		bufferTimeMs = appSettingsLocal.getRtmpIngestBufferTimeMs();
 		dataChannelWebHookURL = appSettingsLocal.getDataChannelWebHook();
+
+		retryLimit = appSettingsLocal.getEndpointRepublishLimit();
```

Comment:
Please give a more descriptive name. `retryLimit` is so generic. It can be something `rtmpEndpointRetryLimit`